search-app/docs/api.md

```diff
@@ -2287,8 +2288,18 @@ Exclude granule by concept id
 Exclude granule by parent concept id
 
     curl "%CMR-ENDPOINT%/granules?provider=PROV1&provider=PROV2&echo_granule_id\[\]=G1000000002-CMR_PROV1&echo_granule_id\[\]=G1000000003-CMR_PROV1&echo_granule_id\[\]=G1000000006-CMR_PROV2&exclude\[concept_id\]\[\]=C1000000001-CMR_PROV2"
+    
+#### <a name="g-include-polygons"></a> Include generated orbital polygons.
 
-#### <a name="sorting-granule-results"></a> Sorting Granule Results
+Include generated polygons in orbital granules. False by default.
```

Comment:
We should make it clear that this flag only apples to ATOM and JSON response formats, not to other formats.